examples/jf-control-app/commands/clusters/ClusterCommand.h

```diff
@@ -0,0 +1,285 @@
+/*
```

Comment:
Are the files in commands/clusters, commands/common, commands/icd, etc actually unique to jf-control-app? 

I'm wondering if these can be included from a common location.

Reply:
Please check the PR description: "jf-control-app is just a duplicate of chip-tool on top of which we are adding specific JF functionality (review of those files is not needed). This duplication is not something new, I also noticed that other features were doing the same for their tools (e.g.: fabric-admin@FabricSync; camera-controller)"

Reply:
Thanks for the suggestion, in the end copied only the files that are unique to the jf-control-app and referenced the untouched files from the examples/chip-tool folder.